pylabrobot/config/__init__.py

```diff
@@ -0,0 +1,91 @@
+"""
+
+Config module. Facilitates reading and writing module-level config files.
+Checks the current directory and all parent directories for a `pylabrobot.ini`
+config file. If the config file does not exist, a default Config object will be
+created and written to the project directory containing the .git directory. If
+this does not exist, the current directory will be used.
+"""
+from pathlib import Path
+from typing import Optional
+
+from pylabrobot.config.config import Config
+from pylabrobot.config.service.file import MultiReader, FileWriter
+from pylabrobot.config.service.ini_file import IniReader, IniWriter
+
+DEFAULT_READER = MultiReader(
+  reader_map={
+    "ini": IniReader(),
+  }
+)
```

Comment:
is `FileReader` missing here?

Reply:
Changed the name of the MultiReader to be MultiFormatFileReader for clarity. Also changing it to DEFAULT_CONFIG_READER